_layouts/docsportal.html

```diff
@@ -13,12 +13,12 @@ <h1>{{ toc.bigheader }}</h1>
   <h5>{{ toc.abstract }}</h5>
   <div id="vendorStrip" class="light-text">
     <ul>
-      <li><a href="/docs/home/" {% if toc.bigheader == "Kubernetes Documentation" %}class="YAH"{% endif %}>HOME</a></li>
+      <li><a href="/docs/home/" {% if toc.bigheader == "Home" %}class="YAH"{% endif %}>HOME</a></li>
```

Comment:
This doesn't seem working. The bigheader for home is currently "About". You may want to revise it and have a try.

Reply:
PR rebased, now it's working